can/util.py

```diff
@@ -289,6 +289,26 @@ def dlc2len(dlc):
     return CAN_FD_DLC[dlc] if dlc <= 15 else 64
 
 
+def channel2int(channel):
+    """Try to convert the channel to an integer.
+
+    :param str channel:
```

Comment:
Maybe we should support arbitrary Python objects here, because the `virtual` and other future interfaces might use anything as an channel identifier. We could just check for type character string, `None`, and integral number, and simply use `channel = str(channel)` or `channel = repr(channel)` for the rest.